src/llama.cpp

```diff
@@ -1478,6 +1480,23 @@ static const std::map<llm_arch, std::map<llm_tensor, std::string>> LLM_TENSOR_NA
             { LLM_TENSOR_FFN_UP,          "blk.%d.ffn_up" },
         },
     },
+    {
+        LLM_ARCH_GRANITE_MOE,
+        {
+            { LLM_TENSOR_TOKEN_EMBD,      "token_embd" },
+            { LLM_TENSOR_OUTPUT_NORM,     "output_norm" },
```

Comment:
```suggestion
            { LLM_TENSOR_OUTPUT_NORM,     "output_norm" },
            { LLM_TENSOR_OUTPUT,          "output" },
```

Necessary for <https://huggingface.co/ibm/PowerMoE-3b>, which does have `output.weight`.

Reply:
Huh! Converting `PowerMoE-3b` has worked fine for me without this. I don't think it hurts to add though and is probably a good future-proof addition for other models that use this architecture as a close cousin to `llama`.

Reply:
This might be because I used the weights from after <https://huggingface.co/ibm/PowerMoE-3b/commit/eac9d00a44c027335d326b62c3f45d11178220e9> which added `lm_head.weight` (aka `output.weight`), while you likely had the model downloaded before that.

Reply:
Ah, spot on! Thanks for the sleuthing.